src/styles.scss

```diff
@@ -0,0 +1,364 @@
+$font: Lucida Sans Unicode;
```

Comment:
I like you used variables - it is the best practices. In huge apps it is mandatory, since the same color can be used by multiple components over the app.

Can we move forward? make scss file `globals.scss`, move all vars there and import it in the current file.

Reply:
Done